localization.zhCN.lua

```diff
@@ -28,5 +28,13 @@ if GetLocale() == "zhCN" then
 	ns.T["events"] = "事件"
 	ns.T["settings"] = "设置"
 	ns.T["about"] = "关于"
-	ns.T["saveLocationHelpText"] = "截图将保存到操作系统的默认位置"
+	ns.T["saveLocationHelpText"] = [[截图将保存到操作系统的默认位置
+
+Windows:  C:\Program Files (x86)\World of Warcraft\_%s_\Screenshots
+MacOS:     \World of Warcraft\_%s_\Screenshots]]
```

Comment:
I think we need to inject the folderName here

```suggestion
	ns.T["saveLocationHelpText"] = [[截图将保存到操作系统的默认位置

Windows:  C:\Program Files (x86)\World of Warcraft\_%s_\Screenshots
MacOS:     \World of Warcraft\_%s_\Screenshots]],
	folderName(),
	folderName()
```

gives me an idea to make this a bit simpler, but this will do for now